microsoft/testsuites/dpdk/dpdktestpmd.py

```diff
@@ -587,7 +587,9 @@ def _load_drivers_for_dpdk(self) -> None:
         else:
             mellanox_drivers = ["mlx5_core", "mlx5_ib"]
         modprobe = self.node.tools[Modprobe]
-        if isinstance(self.node.os, Debian) and not isinstance(self.node.os, Ubuntu):
+        if isinstance(self.node.os, Ubuntu):
+            pass
```

Comment:
Add comments explain like Ubuntu doesn't need prerequisite setup.

Reply:
It does, it just doesn't need any special casing up at the top of the function. I'll comment.